enterprise_gateway/client/gateway_client.py

```diff
@@ -12,7 +12,7 @@
 except ImportError:
     import Queue as queue
 
-DEFAULT_TIMEOUT = 60 * 60
+DEFAULT_TIMEOUT = 60 * 5  # Use 5 minute timeout (should remain < 10 min to avoid travis timeouts
 
```

Comment:
We should not set the default based on Travis, but enable iTests to pass their desired timeout otherwise, this will not work on real use cases where a big data invocation might take a little while to respond.

Reply:
I agree this should be more easily configured.  Upon closer inspection, this timeout is really a connection-oriented timeout and does not correspond to the overall length of a cell's processing.  In fact, its default should be on the order of 60 seconds.  I'll probably reset to 120 just to be on the safe side.

Reply:
In the latest update, this has been changed to `REQUEST_TIMEOUT` (with a default of 120) and can be updated via env `REQUEST_TIMEOUT`.